bin/functions.d/eXist-settings.sh

```diff
@@ -1,6 +1,8 @@
-# -*-Shell-script-*-
-# Common eXist script functions and settings
-# $Id:eXist-settings.sh 7231 2008-01-14 22:33:35Z wolfgang_m $
+#!/usr/bin/env bash
```

Comment:
A wise (?) man (you?) told me once that is this is a dangerous construct... makes the script vulnerable ?

Reply:
Hmmm. I am not certain about that.

However it seems that these days, `#!/usr/bin/env bash` is recommended over `#!/bin/bash` for portability (not all systems have bash in `/bin` e.g. NixOS).